AlicerceTests/Logging/Destinations/Log+StringLogDestination.swift

```diff
@@ -0,0 +1,51 @@
+//
```

Comment:
Do we want this `LogDestination` ? it doesn't seems useful for me, it will use a bunch of memory, so i assume no one will use that 😕